### ISSUE TITLE
README improvement 

### ISSUE THREAD

**Issue by floppy-engineer** [cantino/mcfly#66]
Hi,
Please add "Usage" section to the readme file. And especially information about using '%' symbol as a way to use multiple searching patterns.
Thanks in advance

**Comment by cantino** [comment]
I don't believe that `%` currently allows multiple searches. Were you seeing that behavior?

**Comment by floppy-engineer** [comment]
I kinda see it right now) There is much more OC commands in my history not only "oc get nodes".
McFly 0.3.4 (in Ubuntu flavor in WSL)
![Capture](https://user-images.githubusercontent.com/53353094/66029676-6ea02980-e508-11e9-8e31-73de757a5900.PNG)

**Comment by cantino** [comment]
Hah, good discovery! This is an unintended feature from using sqlite as the database. Cool!
